changelog/fragments/run-bundle-upgrade-bugfix.yaml

```diff
@@ -0,0 +1,25 @@
+# entries is a list of entries to include in
```

Comment:
> For run bundle, when an index image is being passed, we are now generating just the extra FBC needed in the ConfigMap and mounting that as a volume in the registry pod. Sorun bundle-upgrade` needs to be updated to generate just the extra FBC instead of creating the entire declarative config and appending bundle contents to it which will cause duplicate package/channel/bundle errors and fail.

So, we create another FBC index to add the upgraded bundle. Is that the solution?

Reply:
@camilamacedo86 We don't create another FBC index to add the bundle. We add the just the extra bundle contents to a `ConfigMap` under the directory (which is `/configs`) that hosts the index already, so we don't have to append to index's contents again.